onnxruntime/core/providers/mkldnn/nn/conv.h

```diff
@@ -4,18 +4,26 @@
 #pragma once
 #include "core/framework/op_kernel.h"
 #include "core/providers/cpu/nn/conv.h"
+#include "../mkldnn_execution_provider.h"
 
 namespace onnxruntime {
 namespace mkl_dnn {
+
 template <typename T>
 class Conv final : public onnxruntime::Conv<T> {
  public:
   Conv(const OpKernelInfo& info) : onnxruntime::Conv<T>(info) {
+    if (info.GetExecutionProvider()->Type() == kMklDnnExecutionProvider) {
+      provider_ = (const_cast<MKLDNNExecutionProvider*>(
```

Comment:
Why const_cast?

Reply:
I referred to this code to get provider object.

https://github.com/Microsoft/onnxruntime/blob/a78acb2d2ca07f51ce33b3995e9b45bfb691e455/onnxruntime/core/providers/cuda/cuda_common.h#L33